core/src/main/java/org/elasticsearch/plugins/PluginManager.java

```diff
@@ -80,15 +80,16 @@
                     "analysis-phonetic",
                     "analysis-smartcn",
                     "analysis-stempel",
-                    "cloud-aws",
                     "cloud-azure",
                     "cloud-gce",
                     "delete-by-query",
                     "discovery-multicast",
+                    "discovery-ec2",
```

Comment:
I like the fact that it's alphabetically sorted :) so let's put it before multicast?